tests/Rector/Property/ProphecyPHPDocRector/config/configured_rule.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+declare(strict_types=1);
+
+use Rector\Config\RectorConfig;
+use Rector\PHPUnit\Rector\Property\ProphecyPHPDocRector;
+
+return static function (RectorConfig $rectorConfig): void {
+    $rectorConfig->import(__DIR__ . '/../../../../../config/config.php');
+
+    $rectorConfig->rule(ProphecyPHPDocRector::class);
+
+    $rectorConfig->importNames();
+    $rectorConfig->importShortClasses(false);
```

Comment:
By convention, the test should only verify the rule alone, without any configuration.

The imports can be handled in a follow up PR or better in the import rule itself.

Reply:
I just added them to show that they are not working like expected ;).

I repushed and update the tests. If somebody want to take it over to convert it to `Class_` or `Property` listener I'm fine with it, I will not able todo that soon.

I'm still curious about the `use statement` part what is the correct way to add any use statements to a file?

Reply:
See https://github.com/rectorphp/rector-phpunit/pull/114#discussion_r990774677